Library/Formula/poppler.rb

```diff
@@ -52,7 +52,9 @@ def install
       --enable-introspection=yes
     ]
 
-    if build.with? "qt"
+    if build.with?("qt") && build.with?("qt5")
+      odie "poppler: --with-qt and --with-qt5 cannot be used at the same time"
```

Comment:
Maybe could just have `Formula#odie` throw an exception if we really wanted. Would rather not add more to the DSL unless really necessary (and eventually we'll solve the "conflicting options" stuff in a more elegant way, I expect)